Automattic/WP-Job-Manager
# Change string positions to listings / Clarify "Archives" Language in Expired Listings / Filled Positions Settings

Fixes #2491, Fixes #2492

### Changes Proposed in this Pull Request

* Change appearance of string positions to listings

### Testing Instructions

* Go through all screens and check the appearance of positions and check if they are changed to listings after the PR


### Release Notes

* Change positions to listings for consistency
